src/_pytest/_code/code.py

```diff
@@ -802,7 +802,7 @@ def _makepath(self, path):
     def repr_traceback(self, excinfo: ExceptionInfo) -> "ReprTraceback":
         traceback = excinfo.traceback
         if self.tbfilter:
-            traceback = traceback.filter()
+            traceback = traceback.apply_filter()
```

Comment:
We probably shouldn't change this function name; I'm not sure it is public, but doesn't seem worth the risk.

Reply:
sure, I was going to make a follow up issue to track all the overrides we have, so we could always just do it later